front/src/components/Auth/Register.vue

```diff
@@ -4,78 +4,91 @@
       <q-card-section>
         <div class="text-h5">Register</div>
       </q-card-section>
-      <q-card-section class="q-gutter-md">
+
+      <q-card-section class="q-pt-none q-pb-sm q-gutter-sm">
         <template v-if="registrationAnyAllowed">
           <q-input
             v-model="form.login"
             filled
-            label="Choose a login"
+            dense
+            label="Username"
             required
           />
           <password-input
             v-model="form.password"
-            label="Choose a password"
+            label="Password"
+            dense
             clearable
             required
           />
           <q-input
             v-if="!!token"
             filled
+            dense
             readonly
             :model-value="token"
-            hint=""
             label="Auth token"
-          />
+            class="q-pt-sm"
+          >
+            <template #prepend>
+              <q-icon name="key" />
+            </template>
+          </q-input>
           <template v-else>
             <q-checkbox
               v-if="registrationPasswordForced"
               :model-value="true"
               disable
-              title="A password is required to log on this instance."
-              label="I have a password"
+              title="A registration password is required to log on this instance."
+              label="I have a registration password (required)"
             />
             <q-checkbox
               v-else-if="registrationPasswordAllowed"
               v-model="usePassword"
-              label="I have a password"
+              label="I have a registration password"
             />
             <password-input
               v-if="
                 registrationPasswordAllowed &&
                 (usePassword || registrationPasswordForced)
               "
-              v-model="form.ctfnotePassword"
+              v-model="form.registrationPassword"
               filled
+              dense
               clearable
-              label="CTFNote password"
+              label="Registration password"
               required
-            />
+              class="q-mt-xs"
+            >
+              <template #prepend>
+                <q-icon name="key" />
+              </template>
+            </password-input>
           </template>
         </template>
-        <div v-else>Registration are disabled.</div>
+        <div v-else class="q-pb-sm">Registration is disabled.</div>
```

Comment:
Not a regression, but I think this text is not always true: if registration is disabled but registration with password is enabled and you enter the incorrect registration password, you see this text but the text should be something like 'registration password is incorrect'. Can we fix this through frontend or is this a backend issue?

Reply:
This text only appears when you visit `/#/auth/register` when registrations are fully disabled right? I could not get this to appear by entering an incorrect registration password.

Reply:
Ah I think I now understand my confusion: I toggled the admin setting while not reloading the registration page. My bad.